pkg/v1/tkg/client/validate.go

```diff
@@ -813,39 +813,73 @@ func (c *TkgClient) ValidateVsphereVipWorkloadCluster(clusterClient clusterclien
 
 // ValidateVsphereResources validates vsphere resource path specified in tkgconfig
 func (c *TkgClient) ValidateVsphereResources(vcClient vc.Client, dcPath string) error {
+	var path, resourceMoid string
+	var err error
+
 	for _, resourceType := range VsphereResourceType {
```

Comment:
nit, preexisting: VsphereResourceType -> VsphereResourceTypes or VsphereResourceConfigKeys?

Reply:
Done. Thanks for the suggestion.